overrides.d.ts

```diff
@@ -1,8 +1,8 @@
-declare module "*developer-docs-links.yaml" {
-  import { DeveloperDocsLink } from "./src/types"
-  const content: Array<DeveloperDocsLink>
-  export default content
-}
+// declare module "./src/data/developerDocsLinks.json" {
+//   import { DeveloperDocsLink } from "./src/types"
+//   const content: Array<DeveloperDocsLink>
+//   export default content
+// }
```

Comment:
Not removing this yet since I'm planning on reverting this. There is a current issue with Gatsby loading yaml files that was already [fixed](https://github.com/gatsbyjs/gatsby/issues/37453). Once the patch is released I'll update it and revert these changes.